scripts/test.sh

```diff
@@ -9,13 +9,13 @@ CONFIG_FILE_NAME="hardhat.config.ts"
 
 # setup example repo
 rm -rf starknet-hardhat-example
-EXAMPLE_REPO_BRANCH="plugin"
-if [[ "$CIRCLE_BRANCH" == "master" ]] && [[ "$EXAMPLE_REPO_BRANCH" != "plugin" ]]; then
+EXAMPLE_REPO_BRANCH="plugin-declare-return-tx-hash"
```

Comment:
Thanks for appropriately modifying test.sh. You can revert it now that https://github.com/Shard-Labs/starknet-hardhat-example/pull/103 has been merged.

Reply:
Done: 0d5a6c8d8e6f8bc08accf8911281fb5baa3f4a60